src/docs/cookbook/networking/web-sockets.md

```diff
@@ -53,14 +55,14 @@ In this example, use a [`StreamBuilder`][]
 widget to listen for new messages, and a
 [`Text`][] widget to display them.
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (StreamBuilder)" replace="/_channel/channel/g"?>
 ```dart
 StreamBuilder(
-  stream: widget.channel.stream,
```

Comment:
by moving the `channel` variable inside the sample widget state, I removed the need to have the `widget` property here